packages/next/src/ssrPrepass.ts

```diff
@@ -59,7 +59,7 @@ export const ssrPrepass: TRPCPrepassHelper = (opts) => {
       if (typeof parent.ssr === 'function') {
         try {
           return await parent.ssr({ ctx: appOrPageCtx.ctx });
-        } catch (e) {
+        } catch {
```

Comment:
As of https://github.com/eslint/eslint/issues/17974 -> https://github.com/eslint/eslint/pull/18043, these are now flagged by default by `no-unused-vars`. They were unused before.